src/post_section/post_section.jsx

```diff
@@ -30,102 +31,63 @@ function LinksList({ link }) {
         alt={link.name} /></Link>
     </td>;
 }
-class PostSection extends React.Component {
-    constructor(props) {
-        super(props);
-    }
-    render() {
-        return (
-            <div>
-                <section className="second_section">
-                    <div className="second_section_block">
-                        <div className="author_of_post_info">
-                            <div className="inf_about_author">
-                                <img src="/photos/avatar.png" width="56" height="56" alt="avatar" />
-                                <div className="text_about_author">
-                                    <p className="author_name">Mika Matikainen</p>
-                                    <p className="time"> <time datetime="2020-04-15">Apr 15, 2020</time> · 4 min read</p>
-                                </div>
+const PostSection = () => {
+
+    const [data, setPosts] = useState({ posts: [] })
+    useEffect(() => {
+        fetch(`https://dolphin-app-cbjj4.ondigitalocean.app/posts/${index}`)
+            .then(response => response.json())
+            .then(data => setPosts({ posts: data }))
+    })
+    const formattedDate = new Date(data.posts.updated_at).toLocaleDateString('en-US', { year: 'numeric', month: 'short', day: 'numeric' });
+    const paramsString = document.location.href;
+    const index = paramsString.split("/").slice(-1)[0] 
+    return (
+        <div>
+            <section className="second_section">
+                <div className="second_section_block">
+                    <div className="author_of_post_info">
+                        <div className="inf_about_author">
+                            <img className="avatar" src="/photos/avatar.jpg" width="56" height="56" alt="avatar" />
+                            <div className="text_about_author">
+                                <p className="author_name">Misha Matusevich</p>
+                                <p className="time"> <time datetime="YYYY-MM-DD">{formattedDate}</time> · {data.posts.content?Math.trunc(data.posts.content.length/250):"few"} min read</p>
```

Comment:
1. improve the code style. you can use `eslint` or `prettier` or both or format your code using IDE or smth else. learn how to automatically format your code so it doesn't look ugly
```suggestion
                                <p className="time"> <time datetime="YYYY-MM-DD">{formattedDate}</time> · {data.posts.content ? Math.trunc(data.posts.content.length/250) : "few"} min read</p>
```
2. `data.posts.content` is an HTML string, so when you access `data.posts.content.length` it takes into account html tags to calculate reading time. you need to extract text content first and then calculate reading time based on the length of the text